docs/NETWORK-MODELS.md

```diff
@@ -0,0 +1,41 @@
+# Network Models
+
+What are the network models that EVE supports for Edge Container workloads, and how are they implemented?
+
+The first half of this document describes _what_ those network models are and how they impact workload communications. The second half describes _how_ those models are implemented.
+
+## Models
+
+Fundamentally, each Edge Container workload needs to communicate with services outside itself. Those can be one or more of:
+
+* other Edge Container workloads on the same device
+* the network to which the device is attached
```

Comment:
We actually don't have a notion of "the network to which the device is attached"

Reply:
But don't we have the "bridged" option, where it just is connected to an underlying network?